### PR TITLE
docs: Update UG for deletePerson and addTask

### PR DESCRIPTION
**Pull request by pra-navi**
## What type of PR is this? (check all applicable)

-   [ ] Refactor
-   [ ] Feature
-   [ ] Bug Fix
-   [ ] Optimization
-   [X] Documentation Update

## Description

-   Update UG with deletePerson and addTask

## Related Tickets & Documents

-  #58 

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 75% and above for new changes._

-   [ ] Yes
-   [X] No, and this is why: https://docs-developer-guide-v1-1.coordimate.pages.dev/
-   [ ] No, there are UI changes that I'm unable to cover.
-   [ ] I need help with writing tests

### CODE REVIEW

File: docs/DraftUserGuide.md [hunk 1]
```diff
@@ -303,20 +303,18 @@ Output:
 
 ### 8. Adding a task: `addTask`
 
-Adds a person, a vendor, to your contact list.
+Adds a task, to your task list.
```
Comment (jiakai-17) [inline docs/DraftUserGuide.md:306]: Perhaps you could remove this redundant comma?

```suggestion
Adds a task to your task list.
```

**Review by jiakai-17 — CHANGES_REQUESTED** [review]
Thanks for the changes, just a small nit.

File: docs/DraftUserGuide.md (hunk 1 above)
Comment (pra-navi) [reply docs/DraftUserGuide.md]: Thanks for catching that! Just updated the line.

**Review by m1oojv — APPROVED** [review]
LGTM Nice Work

**Pull request merged**